sympy/functions/elementary/miscellaneous.py

```diff
@@ -274,13 +290,22 @@ def root(arg, n, k=0, evaluate=None):
 
 
 def real_root(arg, n=None, evaluate=None):
-    """Return the real nth-root of arg if possible. If n is omitted then
-    all instances of (-n)**(1/odd) will be changed to -n**(1/odd); this
-    will only create a real root of a principal root -- the presence of
-    other factors may cause the result to not be real.
+    """Return the real ``n``'th-root of ``arg`` if possible.
```

Comment:
The style guide says that function parameters should be in italicised like `*n*` rather than verbatim:
https://docs.sympy.org/1.5.1/documentation-style-guide.html#code

Reply:
> The style guide says

Thanks for referring to this. It will take some time to make this a habit.